src/appengine/handlers/cron/bandit_probabilities.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Bandit probabilities cron job. Queries from bigquery for updated probability
```

Comment:
Without much context, "bandit probabilities" may be confusing. Please explain this is running multi-armed bandit experiments for fuzzing strategy selection.